src/net_processing.cpp

```diff
@@ -3158,6 +3162,7 @@ void PeerManagerImpl::ProcessMessage(
         if (pfrom.fSuccessfullyConnected) {
             // Disconnect peers that send SENDADDRV2 message after VERACK; this
             // must be negotiated between VERSION and VERACK.
+            LogPrint(BCLog::NET, "sendaddrv2 received after verack from peer=%d; disconnecting\n", pfrom.GetId());
```

Comment:
20724: missing changes after `if (msg_type == NetMsgType::VERACK)` probably due to missing backports

Reply:
Got it, missing 20146 is the missing one